### PR TITLE
Remove broken drop down features

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description:
Since the dropdown was presenting false information as seen on issue #226 , we have decided to remove the dropdown and only present the accurate information. We are presenting the file that is currently running when running the file.

running
placeholder

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/view/components/microbit/MicrobitSimulator.tsx [hunk 1]
```diff
@@ -81,9 +84,18 @@ export class MicrobitSimulator extends React.Component<any, IState> {
...
-                    running_file: message.state.running_file,
-                });
+                if (this.state.play_button) {
+                    this.setState({
+                        currently_selected_file: message.state.running_file,
+                    });
+                } else {
+                    this.setState({
+                        running_file: message.state.running_file,
+
```
Comment (vandyliu) [inline src/view/components/microbit/MicrobitSimulator.tsx]: nit spacing

**Review by vandyliu — APPROVED** [review]
dope lgtm

File: src/view/constants.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const CONSTANTS = {
         NUMERIC_SIX: "Digit6",
         NUMERIC_SEVEN: "Digit7",
     },
-    NO_FILES_AVAILABLE: "Choose a .py file to run on the Simulator",
+    NO_FILES_AVAILABLE: "Save a .py file to run on the Simulator",
```
Comment (nasadigital) [inline src/view/constants.ts]: Actually just focusing works (which is even better).

Comment (nasadigital) [inline src/view/constants.ts]: Did you test the behavior when you press play and there are no python files open?
For me it crashes and gives out an error and then open a random file?
I think it makes more sense if we just pop up a warning or something similar.

File: src/view/components/microbit/MicrobitSimulator.tsx [hunk 3]
```diff
@@ -100,14 +112,9 @@ export class MicrobitSimulator extends React.Component<any, IState> {
...
-                        label={"file-dropdown"}
-                        styleLabel={"dropdown"}
-                        lastChosen={this.state.running_file}
-                        width={300}
-                        textOptions={this.state.active_editors}
-                        onBlur={this.onSelectFile}
-                    />
+                    {this.state.running_file && this.state.play_button
+                        ? CONSTANTS.CURRENTLY_RUNNING(this.state.running_file)
+                        : null}
```
Comment (nasadigital) [inline src/view/components/microbit/MicrobitSimulator.tsx]: Currently the text just pops out of nowhere and also looks a bit out of place. Perhaps play around with the font style and/or think about different ways to expose this information?

**Review by nasadigital — COMMENTED** [review]
HUFE FAN of this change!
Thanks for implementing this~ 💯

File: src/view/constants.ts (hunk 2 above)
Comment (xnkevinnguyen) [reply src/view/constants.ts]: When I do it, it doesn't crash, but it does give the error and open a random file. This seems to be on dev as well and will be addressed as a separate issue

File: src/view/components/microbit/MicrobitSimulator.tsx (hunk 3 above)
Comment (xnkevinnguyen) [reply src/view/components/microbit/MicrobitSimulator.tsx]: I have made a set a height and a box to address this feedback :)

File: src/view/components/cpx/CpxSimulator.tsx [hunk 4]
```diff
@@ -8,7 +8,6 @@ import { sendMessage } from "../../utils/MessageUtils";
 import "../../styles/Simulator.css";
 import PlayLogo from "../../svgs/play_svg";
 import StopLogo from "../../svgs/stop_svg";
-import Dropdown from "../Dropdown";
```
Comment (nasadigital) [inline src/view/components/cpx/CpxSimulator.tsx:11]: I think you can remove this class since it isn't used anymore.

**Review by nasadigital — APPROVED** [review]
LGTM 💯 💯 💯

**Pull request merged**